From: Dmitry Bogatov <KAction@debian.org>
Date: Mon, 26 Aug 2019 22:02:47 +0000
Subject: Fix spelling error in manpage

---
 insserv.8.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/insserv.8.in
+++ b/insserv.8.in
@@ -271,7 +271,7 @@
 Currently the following options are recognized by insserv:
 .TP
 .BR \-v ,\  \-\-verbose
-Perform operation with more diagnotic messages printed on stderr.
+Perform operation with more diagnostic messages printed on stderr.
 .TP
 .BR \-q ,\  \-\-silent
 Perform operations silently. This blocks warning messages
