Description: Fix spelling errors in insserv(8).
From: Kel Modderman <kel@otaku42.de>
---
Index: insserv/insserv.8.in
===================================================================
--- insserv.orig/insserv.8.in
+++ insserv/insserv.8.in
@@ -126,7 +126,7 @@ these tags is stopped.
 The optional X\-Interactive keyword implies that the script using this
 keyword should be started alone in a concurrent boot configuration
 because it interact with the user at the console.  Only the value
-`true' is recogniced.  All other are ignored.
+`true' is recognised.  All other are ignored.
 .PP
 The optional
 .B X\-Start\-Before
