From: Felipe Sateler <fsateler@debian.org>
Date: Fri, 5 Apr 2019 20:32:42 +0000
Subject: warn about dependencies in ignore mode

Bug-Debian: https://bugs.debian.org/811441

Bug-Debian: https://bugs.debian.org/811441
---
 insserv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/insserv.c
+++ b/insserv.c
@@ -3632,7 +3632,7 @@
 		     * Use information from symbolic link structure to
 		     * check if all services are around for this script.
 		     */
-		    if (isarg && !ignore) {
+		    if (isarg) {
 			boolean ok = true;
 			if (del)
 			    ok = chkdependencies(service);
